spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "spec_helper"
+require "rspec/rails/matchers/active_job"
+
+RSpec.describe "active_job matchers" do
```

Comment:
You can use the skip logic and the feature detection you've used above to skip these on non supported Rails...

e.g. `RSpec.describe "active_job matchers", :skip => !RSpec::Rails::FeatureCheck.has_active_job? do`